src/goto-programs/initialize_goto_model.cpp

```diff
@@ -194,7 +194,14 @@ goto_modelt initialize_goto_model(
 
   if(options.is_set("validate-goto-model"))
   {
-    goto_model.validate(validation_modet::EXCEPTION);
+    goto_model_validation_optionst goto_model_validation_options{
+      goto_model_validation_optionst ::set_optionst::all_false};
+
+    // temporarily disabled until regression tests fixed
```

Comment:
How long is "temporarily" and which regression tests are these?

Reply:
Added comment - will add also to PR description.

    // temporarily disabled until adjustments made for regression tests that
    // have no entry point(e.g. testing json-ui warnings when there
    // is no entry point)

Will be the next PR. I'll start working on it right away.

Reply:
goto models really aren't required to have an entry point; we do want to support goto-cc -c in the long term.